README.md

```diff
@@ -14,6 +14,19 @@ on building killer features and UI.
 - [Blog](http://emberjs.com/blog)
 - [Builds](http://emberjs.com/builds)
 
+
+The Ember.js framework enables developers to build scalable, single-page web applications gracefully and efficiently. It does so by providing a set of useful and easily incorporated functionality, the kind commonly found on dynamic, feature-rich web pages. Useful features and benefits of Ember include:
+
+* **Built-in best practices:** all code conforms to industry-standard best practices
+
+* **Rich feature set:**  Ember-specific features such as the Router and the Ember CLI enhance productivity and functionality   
```

Comment:
I'd maybe focus here on the fact that Ember is "batteries included", and then having a separate point about Ember CLI? Maybe "Integrated development workflow".